extension/lib/magento/customer/addItems.js

```diff
@@ -6,17 +6,17 @@ const MagentoRequest = require('../Request')
  * @param {string} input.token - user token for authentication
  * @param {string} input.wishlistId - id of the wishlist to modify
  * @param {array} input.transformedProducts - user token for authentication
- * @returns {Promise<{productIds: string[]}>}
  */
 module.exports = async (context, input) => {
-  const request = new MagentoRequest(context, input.token)
-
-  const wishlistAddItemEndpointUrl = `${context.config.magentoUrl}/wishlists/${input.wishlistId}/items`
   const wishlistItems = input.transformedProducts.map(({ product }) => product)
 
-  const response = await request.send(wishlistAddItemEndpointUrl, 'addFavorites', 'POST', wishlistItems)
+  if (wishlistItems.length > 0) {
```

Comment:
well, since you voted that you like early returns, then you need to adjust this ^_^